Common/Core/TrackSelection.h

```diff
@@ -112,7 +169,7 @@ class TrackSelection
         return track.itsChi2NCl() <= mMaxChi2PerClusterITS;
 
       case TrackCuts::kITSRefit:
-        return (isRun2 && mRequireITSRefit) ? (track.flags() & o2::aod::track::ITSrefit) : true;
+        return (mRequireITSRefit ? (isRun2 ? (track.flags() & o2::aod::track::ITSrefit) : track.hasITS()) : true);
```

Comment:
@njacazio maybe a similar adjustement can be done also for the `TrackCuts::kTPCRefit` case, asking for the `track.hasTPC()` requirement for Run3 tracks, what do you think?

Reply:
Thanks Mattia, indeed, that's a good point, I implemented it